File: filehandling.py
# Writing to a file
with open('example.txt', 'w') as f:
    f.write('Hello, world!\n')
    f.write('This is a sample text file.\n')

# Reading from a file
with open('example.txt', 'r') as f:
    for line in f:
        print(line.strip())  # Strip newline character

File: filehandling.m
% Writing to a file
fid = fopen('example.txt', 'w');
fprintf(fid, 'Hello, world!\n');
fprintf(fid, 'This is a sample text file.\n');
fclose(fid);

% Reading from a file
fid = fopen('example.txt', 'r');
while ~feof(fid)
    line = fgetl(fid);
    disp(line);
end
fclose(fid);